default.json

```diff
@@ -121,5 +121,5 @@
     "web_link_preview": true,
     "keep_websocket_on": false,
     "mls_support_enabled": true,
-    "encrypt_proteus_storage": false
+    "encrypt_proteus_storage": true
```

Comment:
Will this affect already created proteus clients or only when creating a new one?

Reply:
It will migrate any existing proteus client to use the encrypted storage.